lib/src/controls/navigation/tab_view.dart

```diff
@@ -774,6 +777,15 @@ class Tab with Diagnosticable {
   /// The body of the view attached to this tab
   final Widget body;
 
+  /// The color of the tab.
+  final Color? color;
+
+  /// The color of the selected tab.
+  final Color? selectedColor;
+
+  /// The  outline color of the tab.
+  final Color? outlineColor;
```

Comment:
The documentation for these is vague and incomplete, as well as the properties naming. What do you think of `backgroundColor` and `selectedBackgroundColor` instead of `color` and `selectedColor`, respectively?